libs/blocks/quiz/quiz.js

```diff
@@ -57,45 +60,100 @@ const App = () => {
       const currentflow = userFlow.shift();
       if (!currentflow.length) {
         console.log('No next view so setting select question to empty');
```

Comment:
Just noticing `console.log` which should not be here.

Reply:
Thanks for noticing it. Removed.